### ISSUE TITLE
Clarify support or non-support for multiple aria-live regions on the same page

### ISSUE THREAD

**Issue by CharlesBelov** [w3c/aria#1689]
The [spec for aria-live](https://www.w3.org/TR/wai-aria-1.2/#aria-live) appears to be silent on how a screen reader is expected to respond to a page containing two or more aria-live regions. If there are two such regions that get updated at the same time, are both to be read aloud? If both are marked as assertive, does one interrupt the other?

My experience with VoiceOver is that only one such region currently gets read. [Accessibility Quick Wins: Multiple aria-live on single action, Caveat!](https://gaurav5430.medium.com/quick-accessibility-wins-multiple-aria-live-on-single-action-caveat-b79a6f41e7cc) also says this doesn't work.

It would be good if the spec were explicit either that this is outside spec, or is inside and how it should be handled.

**Comment by JAWS-test** [comment]
I agree with you that the specification could be more specific about what should happen when multiple live regions are updated at the same time. However, the behavior you observed seems to be a bug of VoiceOver, because with JAWS and NVDA, even multiple live regions that are updated at the same time are output properly one after the other.

**Comment by brennanyoung** [comment]
Given that some automated audit tools report multiple live regions as an error (or flag it with a warning), I'd agree with the need for clarity here, since nothing in the ARIA spec actually forbids it, and when people see those warnings, they're going to want an explanation. 

In particular, component-based development seeks 'self-contained' solutions (i.e. components are likely to maintain their own live region, rather than relying on a global one), so I don't think a spec which forbids multiple live regions is going to go down well unless it takes account of that dev methodology.

BTW I have also experienced the VO issue where live regions upstage each other completely. They can also 'steal' the announcement of titles of documents loaded into iframes, which seems related. Messing around with "assertive" vs. "polite" does not help. I can confirm that NVDA or Jaws don't have this problem.

It seems to be (at least partly) an 'accessibility supported' issue on given configurations because, as the spec says

> User agents or assistive technologies MAY choose to clear queued changes when an assertive change occurs. (e.g., changes in an assertive region may remove all currently queued changes)

So, even though multiple live regions are not mentioned, it's up to Apple to do that, if they want. I'm not convinced this liberty is ok, because at least some of the other major AT vendors have made a different choice.

If the AT is **not** designed to skip or 'override' the older announcements, what should happen to them? Are they lost? Should a buffer be explicitly specified? (It occurs to me that a global live-region buffer might usefully be provided by the UA).

We've also experienced problems (on NVDA - which maintains a buffer) with buffered announcements coming 'too late' to be relevant (and actually misleading) because some other announcement came first. This is especially problematic for RIAs that have an Essential Exception for complying with WCAG SC 2.2.1: Timing Adjustable.

Should it really be the AT vendors that make the choice to push or pop? Both approaches (buffering or clearing pending announcements) present problems which might be solved by allowing authors some control over how these 'upstaged' announcements are handled.

**Comment by JAWS-test** [comment]
Related: https://github.com/FreedomScientific/VFO-standards-support/issues/285
